docs/tutorial-model-and-deploy.md

```diff
@@ -79,12 +80,40 @@ Click on a *Node Template* to edit its *Properties* and to create new *Relations
 Next we will connect the nodes with *Relationship Templates* to define the dependencies and deployment order of the nodes.
 
 Connect all node templates in the topology with "HostedOn" relationship templates.
-Start with the QHAnaUI_w1 node template that is "HostedOn" the NGINX_latest-w1 node template that is "HostedOn" the DockerContainer_w1 node template that is "HostedOn" the DockerEngine_w1 node template.
+Start with the Shor-UI_w1 node template that is "HostedOn" the NGINX_latest-w1 node template that is "HostedOn" the DockerContainer_w1 node template that is "HostedOn" the DockerEngine_w1 node template.
 The different *relationship types* are explained in the glossary.
 
 ![topology with connected node templates](./images/topology-nodes-with-relationships.png)
 
 
+We also need to model the actual shor service.
+For this we need the following additional nodes:
+
+| Node | Namespace | Explanation |
+|:-----|:----------|:------------|
+| DockerContainer_w1 | `http://opentosca.org/nodetypes` | The docker container hosting the Shor code. |
+| Python_3-w1 | `http://opentosca.org/nodetypes` | The python interpreter running the shor algorithm. |
+| Qiskit_latest-w1 | `https://ust-quantil.github.io/nodetypes` | The SDK required to run the shor algorithm. |
+| Qiskit-App_w1 | `https://ust-quantil.github.io/nodetypes` | The Shor algorithm. |
+| IBMQ_w1 | `https://ust-quantil.github.io/nodetypes` | The proxy node for the connection to the IBMQ cloud. |
+
+![complete topology with all node templates](./images/topology-nodes-complete.png)
+
+Now connect all the nodes accordingly:
+
+| Node | Connection Type | Target Node | Comment |
+|:-----|:---------------:|:------------|:--------|
+| Python_3-w1 | HostedOn | DockerContainer_w1 |  |
+| Qiskit_latest-w1 | HostedOn | DockerContainer_w1 |  |
+| Qiskit_latest-w1 | DependsOn | Python_3-w1 | Qiskit must be installed after python |
+| Qiskit_latest-w1 | HostedOn | IBMQ_w1 | Quiskit circuits are executed on the IBMQ infrastructure. See also [TOSCA4QC: Two Modeling Styles for TOSCA to Automate the Deployment and Orchestration of Quantum Applications](https://doi.org/10.1109/EDOC49727.2020.00024) for more information. |
+| Qiskit-App_w1 | HostedOn | Qiskit_latest-w1 | Qiskit_latest-w1 already depends on python and IBMQ so there is no need for additional connections on this node. |
+| Shor-UI_w1 | ConnectsTo | Qiskit-App_w1 | Models the connection between the shor UI and the backend service. |
+
+![full topology with all node templates connected](./images/topology-nodes-complete-with-relationships.png)
+
+:information_source: Save the changes made so far in the topology editor.
```

Comment:
Consistently either topology modeler or editor (but I guess its modeler @lharzenetter)

Reply:
Yes, please use modeler.

Reply:
Changed, except for the "Open Editor" button in the winery that opens the modeler.